src/endpoints/users.py

```diff
@@ -9,13 +9,13 @@
 
 
 @router.post("/api/users")
-async def create_user(user: schemas.UserCreate, db: orm.Session = fastapi.Depends(services.get_db)):
-    db_user = await services.get_user_by_email(user.email, db)
+async def create_user(user_: schemas.UserCreate, db: orm.Session = fastapi.Depends(services.get_db)):
+    db_user = await services.get_user_by_email(user_.email, db)
 
     if db_user:
         raise fastapi.HTTPException(status_code=400, detail="Email already created")
 
-    user = await services.create_user(user, db)
+    user = await services.create_user(user_, db)
```

Comment:
a bit confusing what is the difference between `user` and `user_`. Maybe it makes sense to use another name here?